core/build.gradle

```diff
@@ -8,11 +8,9 @@ shadowJar {
 
 dependencies {
     api dependencyVersion("slf4j")
-    api dependencyVersion("reactive.streams")
```

Comment:
Are you reliant on this being brought in transitively through rxjava2? Can you envision a scenario where removing this would break an app?

Reply:
I added it as a dependency of runtime and http so don’t believe from a user perspective this will break anything